### DISCUSSION TITLE
Correct settings to use Lighting models?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [lllyasviel/Fooocus#2419] [Q&A] [answered]
These are correct?

![Screenshot_22](https://github.com/lllyasviel/Fooocus/assets/13344308/b4ad3133-af0d-4311-88fa-b9e7b93a3da7)
![Screenshot_21](https://github.com/lllyasviel/Fooocus/assets/13344308/d76ba91a-e418-4a65-b813-ef25f9e90a23)
![Screenshot_20](https://github.com/lllyasviel/Fooocus/assets/13344308/2e245009-1127-4640-b0f2-15eb45fcdea9)
![Screenshot_19](https://github.com/lllyasviel/Fooocus/assets/13344308/d0645257-c100-438b-a832-147fd477e468)

Thx in advance.

**Comment by mashb1t** [comment, marked as answer]
Different samplers and schedulers are recommended, see https://github.com/lllyasviel/Fooocus/discussions/2318 and https://github.com/lllyasviel/Fooocus/pull/2415
